This is the *FULL* output of the test command as it currently stands:
```
$ ./gradlew clean test

> Task :CivModCore:compileJava
/home/Protonull/Desktop/CivModCore/src/main/java/org/bukkit/pseudo/PseudoServer.java:852: warning: [removal] createVanillaChunkData(World,int,int) in Server has been deprecated and marked for removal
        public ChunkGenerator.@NotNull ChunkData createVanillaChunkData(
                                                 ^
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
1 warning

BUILD SUCCESSFUL in 10s
11 actionable tasks: 5 executed, 6 up-to-date
```

--- a/src/main/kotlin/net/civmc/civgradle/common/PlatformCommon.kt
+++ b/src/main/kotlin/net/civmc/civgradle/common/PlatformCommon.kt
@@ -63,7 +63,7 @@ object PlatformCommon {
         project.tasks.withType(Test::class.java) { testing ->
             testing.useJUnitPlatform()
             testing.testLogging { logging ->
-                logging.events(TestLogEvent.values())
+                logging.events(*TestLogEvent.values())
                 logging.exceptionFormat = TestExceptionFormat.FULL
                 logging.showCauses = true
                 logging.showExceptions = true

